arm/Microsoft.Insights/components/deploy.bicep

```diff
@@ -67,8 +67,17 @@ module appInsights_rbac '.bicep/nested_rbac.bicep' = [for (roleAssignment, index
   }
 }]
 
-output appInsightsName string = appInsightsName
+@description('The name of the application insights component')
+output appInsightsName string = appInsights.name
+
+@description('The resource ID of the application insights component')
 output appInsightsResourceId string = appInsights.id
+
+@description('The resource group the application insights component was deployed into')
 output appInsightsResourceGroup string = resourceGroup().name
+
+@description('The instrumentation key of the application insights component')
 output appInsightsKey string = appInsights.properties.InstrumentationKey
+
+@description('The application ID of the application insights component')
```

Comment:
These can be accessed through the `existing` keyword in the resource using them and then through dotted access like `<appInsightsResource>.properties.<whatever>`. I'm leveraging this in the web sites refactoring. I suggest to remove those outputs.

Reply:
removed the key output